### DISCUSSION TITLE
How to align switch with the text + change the colour of switch when it is checked

### DISCUSSION THREAD

**Discussion by Josborne123** [TomSchimansky/CustomTkinter#539] [Q&A]
For some reason when using CTkSwitch the actual switch part is not aligned with the text part, it is slightly above. I was wondering how to fix this. I've tried to use `anchor="center"` but it doesn't change anything

![screenshot1](https://user-images.githubusercontent.com/50294428/196990519-c1b3d51c-309d-4b77-af12-c68618835093.png)

My code for the switch is
`
switch = customtkinter.CTkSwitch(window, text="Mute", command=mute_switch, text_font=("Comic Sans MS bold", 11), bg_color="#e88b2e", button_color="#6c43cd", fg_color="#b8d2d4", button_hover_color="#a589e8")
switch.place(relx=0.11, rely=0.21, anchor="center")
`

Also how do I change the blue colour of the switch when the switch is on to another colour? (shown below)
![screenshot2](https://user-images.githubusercontent.com/50294428/196990481-933bfe69-a645-49ff-a696-2013d909a7f7.png)

**Comment by Akascape** [comment]
@Josborne123 I guess it's due to the font, either change the font or increase the size of CTkSwitch. Eg: `height=25, width=45`

>Also how do I change the blue colour of the switch when the switch is on to another colour? (shown below)

You can add this in your `mute_switch` function to change the button color:
```python
if switch.get()==1:# On value
    switch.configure(button_color="red") # new color
else:
    switch.configure(button_color="#6c43cd") # old color
```

**Reply by Josborne123** [reply]
@Akascape Thanks for you help with the first part. For the second part I meant to ask how to change the colour of the blue bit of the switch (where the arrow is pointing in this image)
![switch](https://user-images.githubusercontent.com/50294428/197182986-4ce86656-5e3c-418b-84de-24c89f508a85.png)
There doesn't seem to be an option to change this. Using fg_color changes the color of the arrowed bit but only when the switch is unchecked.

**Reply by TomSchimansky** [reply]
With ```progress_color``` you can change the color of the switch when its activated.

**Reply by Josborne123** [reply]
Thanks
